examples-next/graphql-api-endpoint/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "keystone-next-app",
+  "name": "keystone-app",
```

Comment:
I don't think we want to change this package.

Reply:
Reverted.